docs/capabilities.md

```diff
@@ -162,3 +162,4 @@
 * `download-call-participants` - Whether the endpoints for moderators to download the call participants is available
 * `config => call => start-without-media` (local) - Boolean, whether media should be disabled when starting or joining a conversation
 * `config => call => max-duration` - Integer, maximum call duration in seconds. Please note that this should only be used with system cron and with a reasonable high value, due to the expended duration until the background job ran.
+* `config => call => blur-background` (local) - Boolean, whether blur background is set by default when joining a conversation
```

Comment:
I'm afraid we could misinterpet at some point:
- blur-background (from Talk) for video input
- background-blur (from server) for general CSS blur filter

```suggestion
* `config => call => blur-virtual-background` (local) - Boolean, whether blur background is set by default when joining a conversation
```